### ISSUE TITLE
Missing menu option to export single chat history if it's a group with topics

### ISSUE THREAD

**Issue by dzek69** [telegramdesktop/tdesktop#27322]
### Steps to reproduce

1. Find any group with topics
2. Open it
3. Click the dots menu

### Expected behaviour

1. There should be an "Export chat history" entry, like it's in every other chat (private chat/private or public group without topics/channel)

### Actual behaviour

"Export chat history" is missing

### Operating system

Linux, KDE Neon

### Version of Telegram Desktop

4.14.3

### Installation source

Static binary from official website

**Comment by john-preston** [comment]
@dzek69 If you switch to "View as Messages" and look in the three dot menu there, in all-messages-across-all-topica history, will export be there?

**Comment by dzek69** [comment]
If i switch to View as Message the option is indeed there.

**Comment by dzek69** [comment]
This option is still missing as of 5.2.2

**Comment by amagnolo** [comment]
> If i switch to View as Messages the option to export is indeed there.

unfortunately, in the export the messages will be all mixed with no attribute to distinguish which topic they belong to

**Comment by gwpl** [comment]
> Unfortunately, when exporting this way, the messages will be mixed together without any way to identify their original topics, thus making them unintelligible

Exactly! If there would be way how to distinguish, one could make a script to post process export, but like this not really - unless I missed sth ofc.

**Comment by 23rd** [comment]
Can be closed?

**Comment by amagnolo** [comment]
> Can be closed?

No, until it will be solved.

**Comment by 23rd** [comment]
> No, until it will be solved.

You can export history of individual topics, what else do you need?

**Comment by amagnolo** [comment]
> You can export history of individual topics, what else do you need?

I am sorry, I didn't notice that the feature is now present in version 6.3.9. 
You can close this issue, IMO.

**Comment by dzek69** [comment]
To me this is still unresolved, the initial report of missing menu entry.

<img width="1128" height="648" alt="Image" src="https://github.com/user-attachments/assets/d3ebb40d-de50-40ff-9c5d-f57fed2dff40" />

Both these menus does not include an option to export the chat. Yes, there is an IMO hidden option to export, but it's super counter intuitive.

Telegram in most cases has superior UX to most of the apps (not only IMs) so I don't see a reason to hide option in the topics view (the sensible default way to browse groups with topics) from the users. Can we add this menu entry?

**Comment by dzek69** [comment]
still valid

**Comment by vladkrcan** [comment]
Still valid
